graph/queries/getUserVotingAndStakingDetails.ts

```diff
@@ -0,0 +1,78 @@
+import graphEndpoint from "constants/graphEndpoint";
+import request, { gql } from "graphql-request";
+import { bigNumberFromFloatString } from "helpers/formatNumber";
+import { UserDataQuery, UserDataT, VoteHistoryByKeyT, VoteHistoryT } from "types/global";
+
+export async function getUserVotingAndStakingDetails(address: string | undefined) {
+  const userDataQuery = gql`
+    {
+      users(where: { address: "${address}" }) {
+        annualPercentageReturn
+        annualReturn
+        cumulativeCalculatedSlash
+        cumulativeCalculatedSlashPercentage
+        countReveals
+        countNoVotes
+        countWrongVotes
+        countCorrectVotes
+        votesSlashed {
+          request {
+            id
+          }
+          voted
+          correctness
+          slashAmount
+          staking
+        }
+      }
+    }
+  `;
+
+  const result = await request<UserDataQuery>(graphEndpoint, userDataQuery);
+  const userData = parseUserVotingAndStakingDetails(result.users[0]);
+
+  return userData;
+}
+
+function parseUserVotingAndStakingDetails(rawUserData: UserDataQuery["users"][0]) {
+  const apr = bigNumberFromFloatString(rawUserData?.annualPercentageReturn);
+  const countReveals = bigNumberFromFloatString(rawUserData?.countReveals);
```

Comment:
Nothing fancy here, just using a helper to parse these float strings into the ubiquitous `BigNumber` type